samples/QmlArExample/qml/main.qml

```diff
@@ -56,6 +56,9 @@ ApplicationWindow {
         onStartTrackingClicked: arcGISArView.startTracking();
         onStopTrackingClicked: arcGISArView.stopTracking();
         onResetTrackingClicked: arcGISArView.resetTracking();
+        onIgnoreTrackingClicked: arSample.setTrackingMode(0);
+        onInitialTrackingClicked: arSample.setTrackingMode(1);
+        onContinuousTrackingClicked: arSample.setTrackingMode(2);
```

Comment:
Ditto: this enum needs to be registered and used here. This is not great for code readability to use hard-coded int values like this.